src/utils/addresses.ts

```diff
@@ -52,3 +52,14 @@ export const parsePrefixedAddress = (value: string): PrefixedAddress => {
 export const formatPrefixedAddress = (address: string, prefix?: string): string => {
   return prefix ? `${prefix}:${address}` : address
 }
+
+export const cleanInputValue = (value: string): string => {
+  const regex = new RegExp('(?:([a-z0-9]+):)?(0x[a-f0-9]{40})\\b', 'i')
```

Comment:
Why this syntax and not just
```suggestion
  const regex = /(?:([a-z0-9]+):)?(0x[a-f0-9]{40})\b/i
```
?